.gitmodules

```
@@ -10,3 +10,7 @@
 	path = stock-logistics-warehouse
 	url = git@github.com:Vauxoo/stock-logistics-warehouse.git
 	branch = 12.0
+[submodule "server-ux"]
+	path = server-ux
+	url = git@github.com:OCA/server-ux.git
```

Use the one at Vauxoo

```suggestion
	url = git@github.com:Vauxoo/server-ux.git
```